src/wagtail_live/models.py

```diff
@@ -157,8 +165,10 @@ def get_updates_since(self, last_update_ts):
             and the updated posts since last_update_ts.
         """
 
+        # Reverse posts list so that latest updates are processed later by the client side.
+        posts = reversed(self.live_posts)
```

Comment:
I'm not sure I understand this. This means that later posts are inserted earlier in the list right? So if the list is processed from start to end, the latest updates are processed earlier this way and not later?

Reply:
LivePageMixin.live_posts keeps posts sorted by -created; latest posts are in the start of the list.
If we reverse the list, latest posts will be on the end of the list.

Therefore, if the list is processed from start to end, latest posts will be processed later.

Does it make sense?

Reply:
Ah, that I overlooked. All right :-)